test/suites/standard2/configuration.robot

```diff
@@ -141,37 +142,80 @@ Deploy MicroShift Without CSI Snapshotter
     ...    Remove Storage Drop In Config
     ...    Restart MicroShift
 
-Custom TLS 1_3 configuration
-    [Documentation]    Configure API server to use TLS 1.3 and verify only that
-    ...    version works
-    [Setup]    Setup TLS Configuration    ${TLS_13_MIN_VERSION}
+Custom TLS 1_2 configuration
+    [Documentation]    Configure a custom cipher suite using TLSv1.2 as min version and verify it is used
+    [Setup]    Setup TLS Configuration    ${TLS_12_CUSTOM_CIPHER}
 
-    ${rc}=    Execute Command
-    ...    openssl s_client -connect ${USHIFT_HOST}:6443 -tls1_3 <<< "Q"
-    ...    sudo=True    return_stdout=False    return_stderr=False    return_rc=True
+    ${config}=    Show Config    effective
+    Should Be Equal    ${config.apiServer.tls.minVersion}    VersionTLS12
+    Length Should Be    ${config.apiServer.tls.cipherSuites}    2
+    Should Contain    ${config.apiServer.tls.cipherSuites}    TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256
+    Should Contain    ${config.apiServer.tls.cipherSuites}    TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+
+    # custom cipher defined for this test
+    ${stdout}    ${rc}=    Openssl Connect Command
+    ...    ${USHIFT_HOST}:6443
+    ...    -tls1_2 -cipher ECDHE-RSA-CHACHA20-POLY1305
```

Comment:
Given the different syntax between IANA and openssl I think a comment could help. Not a showstopper though.

Reply:
thanks, addressed in the last commit